typings/single-spa.d.ts

```diff
@@ -0,0 +1,99 @@
+declare module "single-spa" {
+  // ./start.js
+  export function start(): void;
+  export function isStarted(): boolean;
+
+  // ./jquery-support.js
+  export function ensureJQuerySupport(jQuery?: JQuery): void;
+
+  // ./applications/timeouts.js
+  export function setBootstrapMaxTime(
+    time: number,
+    dieOnTimeout?: boolean
+  ): void;
+  export function setMountMaxTime(time: number, dieOnTimeout?: boolean): void;
+  export function setUnmountMaxTime(time: number, dieOnTimeout?: boolean): void;
+  export function setUnloadMaxTime(time: number, dieOnTimeout?: boolean): void;
+
+  type timeoutConfig = {
+    millis: number;
+    dieOnTimeout: boolean;
+  };
+
+  type globalTimeoutConfig = {
+    bootstrap: timeoutConfig;
+    mount: timeoutConfig;
+    unmount: timeoutConfig;
+    unload: timeoutConfig;
+  };
+
+  export function reasonableTime(
```

Comment:
This function is not exported from single-spa. https://github.com/CanopyTax/single-spa/blob/master/src/single-spa.js

Reply:
You're right, removed